docs/AWS/getting-started-AWS.md

```diff
@@ -23,9 +23,9 @@ python3 -m virtualenv ~/apc-ve
 source ~/apc-ve/bin/activate
 ```
 
-Then install ParallelCluster. If you do not install the correct current version, 3.5.1, then you can not launch the AMI which is made with that version.
+Then install ParallelCluster. If the version of ParallelCluster does not match the version used to genearte the AMI then the cluster creation operation will fail. Currently E4S-Pro AMIs are built with version 3.8.0 but you can check the name of the image for the ParallelCluster version.

```

Comment:
```suggestion
Then install ParallelCluster. If the version of ParallelCluster does not match the version used to generate the AMI then the cluster creation operation will fail. Currently E4S-Pro AMIs are built with version 3.8.0 but you can check the name of the image for the ParallelCluster version.

```

Reply:
Regarding checking the version, I'm not sure that the full name is visible in the Marketplace — I think it just gives them the AMI ID when they subscribe. So we should possibly tell them to check the description in the marketplace instead. We may also want a table (here or or on its own page) saying what version of pcluster is used with each version of the image.

Reply:
Also here, we're inconsistent in how we refer to the product. Here we have "E4S-Pro" with a hyphen, which I don't think is correct, although we're going to be changing the product name so this isn't urgent.

Reply:
The version string is visible in both the AMI search and the product page so I think we're good to go there. The available AMI's are only built with 3.8.1 right now but I'll add a table when we start having more to keep track of.